src/web/index.js

```diff
@@ -17,6 +17,7 @@ import {
   themeChangeActions
 } from '../lib/store';
 import App from './lib/components/App';
+import * as storage from './lib/storage';
```

Comment:
Might just be the Pythonista in me, but `*` imports kind of stink. Any chance you could `export default { init, etc }` from `lib/storage` instead?

Reply:
I kind of don't like the repetition in listing all the exports a second time, but I also don't super like the `*` import either. I'll go with the latter though and tweak it